rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] [ OPTIONAL] It is required to have `:person, :book` inside `attr_accessor`. 
- **Requirement** `attr_accessor :date, :book, :person`